legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/api/GitLabEntityApiTestResource.java

```diff
@@ -0,0 +1,159 @@
+// Copyright 2021 Goldman Sachs
```

Comment:
It looks like you maybe copied this from an older version of IntegrationTestGitLabEntityApis. Perhaps you should rebase and then do the move again?

Reply:
Oops butter fingered when git stashing. Fixed!